### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qkrybin**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [x] others (anything other than above)

---

**Related:** 
- https://github.com/dsrvlabs/vatz/issues/470

**Summary**
- Update to the latest version

---

can't fix every case for now but need to categorize error message for later as much as possible.

### CODE REVIEW

**Review by heejin-github — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -13,55 +13,61 @@
...
+We're on a mission to both transform the way people experience blockchain technology, and help them shape it. 
+As validators, we provide tools for low-cost and low-effort node management for anyone wanting to onboard next-generation blockchain technology.
 
 --- 
 # Usage of VATZ
 
-## How to get started with **VATZ**?
-Please follow [Installation guide](docs/installation.md) to install and start VATZ.
+## How to get started with VATZ
+Check out the[Installation Guide](docs/installation.md) to install and start using VATZ.
```
Comment (meetrick) [inline README.md:56]: How about modifying the documentation link like this?
`docs/installation.md` -> `./docs/installation.md`
image

File: README.md [hunk 2]
```diff
@@ -13,55 +13,61 @@
...
 
-## How to use **VATZ** CLIs?
-Please check [VATZ CLIs guide](./docs/cli.md) to find available CLI arguments.
+## How to use **VATZ** CLIs
+Refer to the [VATZ CLIs guide](./docs/cli.md) to find available CLI arguments.
```
Comment (meetrick) [inline README.md:59]: How about modifying the documentation link like this?
`./docs/cli.md -> docs/cli.md`
image

**Review by meetrick — COMMENTED** [review]
Please check my comment!

File: README.md (hunk 2 above)
Comment (qkrybin) [reply README.md]: changed!

**Review by gnongs — APPROVED** [review]
LGTM!

**Review by xellos00 — APPROVED** [review]

**Pull request merged**